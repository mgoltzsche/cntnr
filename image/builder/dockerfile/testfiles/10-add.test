FROM alpine:3.7
ADD entrypoint.sh /entrypoint.sh
ADD --chown=root:root entrypoint.sh *.conf /
ADD http://example.org/index.html /var/www/html/

# ASSERT RUN [ -f "/entrypoint.sh" ] && [ -f "/cfg-a.conf" ] && [ -f "/var/www/html/index.html" ]

# TODO: test different users (need to run test as root or use proot)