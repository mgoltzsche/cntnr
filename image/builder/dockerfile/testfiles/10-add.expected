FROM alpine:3.7
ADD dir="./ctx" "entrypoint.sh" "/entrypoint.sh" 0:0
ADD dir="./ctx" "entrypoint.sh" "*.conf" "/" root:root
ADD dir="./ctx" "http://example.org/index.html" "/var/www/html/" 0:0